app/controllers/v0/sessions_controller.rb

```diff
@@ -33,6 +33,9 @@ def saml_callback
       if @saml_response.is_valid? && persist_session_and_user
         async_create_evss_account(@current_user)
         redirect_to SAML_CONFIG['relay'] + '?token=' + @session.token
+
+        obscure_token = Session.obscure_token(@session.token)
+        Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
```

Comment:
We need the functionality in order to comply with auditing requirements. Meeting the requirement of tracking across all sessions requires a single log entry per login, so why should we not do it?

Logging the obscure_token in the `Rails.logger.info...` line is not redundant. On that request, the browser doesn't yet have the token, so it will not be included with the request and the log_tag thing won't be able to log it. The next request from the browser will contain the token, but at that point it's too late to connect it back to the UUID.